Shared/ApiShared.cs

```diff
@@ -13,7 +13,7 @@
     using System.Threading.Tasks;
     using CloudinaryDotNet.Actions;
     using Newtonsoft.Json;
-
+

```

Comment:
Please make sure to use linux line ending(LF) and not windows(CRLF) and please do not mix them.